lib/install/fsm.go

```diff
@@ -248,12 +247,10 @@ func (f *fsmEngine) GetExecutor(p fsm.ExecutorParams, remote fsm.Remote) (fsm.Ph
 
 // RunCommand executes the phase specified by params on the specified server
 // using the provided runner
-func (f *fsmEngine) RunCommand(ctx context.Context, runner fsm.RemoteRunner, server storage.Server, p fsm.Params) error {
-	args := []string{"install", "--phase", p.PhaseID, fmt.Sprintf("--force=%v", p.Force)}
-	if f.RemoteOpsURL != "" && f.RemoteOpsToken != "" {
-		args = append(args,
-			fmt.Sprintf("--ops-url=%v", f.RemoteOpsURL),
-			fmt.Sprintf("--ops-token=%v", f.RemoteOpsToken))
+func (f *fsmEngine) RunCommand(ctx context.Context, runner rpc.RemoteRunner, server storage.Server, p fsm.Params) error {
+	args := []string{"plan", "execute", "--phase", p.PhaseID}
+	if p.Force {
+		args = append(args, "--force")
```

Comment:
If I remember correctly, this change alone broke etcd upgrades so the etcd plan had to be tweaked to fix it. Should be taken into account if not yet.

Reply:
Discussed offline that the necessary changes are already present in 5.5 so we should be ok.